external-crates/move/crates/move-compiler/tests/move_check/naming/named_blocks_invalid.snap

```diff
@@ -75,6 +75,15 @@ error[E13001]: feature is not supported in specified edition
    │
    = You can update the edition in the 'Move.toml', or via command line flag if invoking the compiler directly.
 
+error[E01002]: unexpected token
```

Comment:
Can you explain these error changes? They seem very bad to me.

Reply:
This one was quite unexpected as the change I made should not have affected this test case as there is not actual parsing error there. However, `consume_token` function was actually returning an error, which did lead to modified behavior, but not a parsing-related error but edition-related error. My take on this is that we should abandon an increased resilience attempt when observing edition-related error and (revert to non-resilient case when this type of error is seen)